atom/common/api/atom_bindings.h

```diff
@@ -11,6 +11,9 @@
 #include "base/strings/string16.h"
 #include "v8/include/v8.h"
 #include "vendor/node/deps/uv/include/uv.h"
+#include "atom/common/crash_reporter/crash_reporter.h"
+
+using crash_reporter::CrashReporter;
```

Comment:
Usually you don't want to add `using` to header files like this. Just use the explicit scope

 I don't know the atom style guides but you may also want to replace the using statements in the implementation files too.

Also, if you can, avoid adding the new include and instead use a forward declaration of the required types